Formula/libmng.rb

```diff
@@ -4,6 +4,7 @@ class Libmng < Formula
   url "https://downloads.sourceforge.net/project/libmng/libmng-devel/2.0.3/libmng-2.0.3.tar.gz"
```

Comment:
Homepage https://libmng.com/ is TLS 1.3-only. I think only Monterey curl supports TLS 1.3. Some options, but each has a demerit:
- Just ignore audit failures. Not the ideal scenario since it doesn't allow clean CI pass until Monterey is oldest OS.
- Could add `:homebrew_curl`, but seems excessive for just homepage audit fix.
- Could change homepage to SourceForge as current tarball is a fork. However, the original URL is more common on Repology.

Reply:
For now, will add `:homebrew_curl` and add a comment on reason.